pyflakes/test/test_other.py

```diff
@@ -161,6 +161,15 @@ def _(): pass
         def _(): pass
         ''')
 
+    def test_redefinedUnderscoreImportation(self):
+        """
+        Test that shadowing an underscore importation raises a warning.
+        """
+        self.flakes('''
+        from .i18n import _
+        def _(): pass
+        ''', m.RedefinedWhileUnused)
+
```

Comment:
Isn't it already tested here?
https://github.com/eight04/pyflakes/blob/a11599a8bd25bc9c9ab680b5849f50af6b279222/pyflakes/test/test_imports.py#L281